Remotion/Web/Development.WebTesting/WebDriver/Configuration/Firefox/FirefoxConfiguration.cs

```diff
@@ -96,8 +96,10 @@ public FirefoxConfiguration ([NotNull] WebTestConfigurationSection webTestConfig
     public override ScreenshotTooltipStyle TooltipStyle => ScreenshotTooltipStyle.Firefox;
 
     /// <inheritdoc />
-    public virtual FirefoxOptions CreateFirefoxOptions ()
+    public virtual FirefoxOptions CreateFirefoxOptions (DriverConfiguration driverConfiguration)
```

Comment:
passing the driverconfig as a parameter feels strange to me. It mixes the abtraction levels. The change is also missing the aspect that we can specify the headless default via WebTestConfig which is then supplied to the FirefoxConfig but ignored.